github-merge.py

```diff
@@ -198,6 +198,9 @@ def print_merge_details(pull, title, branch, base_branch, head_branch, acks):
                 print('* {} {}({}){}'.format(message, ATTR_NAME, name, ATTR_RESET))
         else:
             print('{}Top commit has no ACKs!{}'.format(ATTR_WARN, ATTR_RESET))
+    if message is not None:
```

Comment:
nit, one line:
```py
if message is not None and '@' in message:
```

Reply:
I had done it to match above, but sure, changed.